src/location-search/LocationSearch.tsx

```diff
@@ -182,7 +180,7 @@ const LocationSearch: React.FC<Props> = ({
         </View>
       </View>
 
-      {true && (
+      {error && (
```

Comment:
How did this even pass .... Oh well

Reply:
It was me updating the code from your code review and pushing silly code. Figured it didn't get reviewed after the last commit.

Reply:
No, didn't see through changes past my last comment I think :woozy_face: